### ISSUE TITLE
[Question] looking to hire a widget developer

### ISSUE THREAD

**Issue by thucar** [thingsboard/thingsboard#4847]
**Component**
 * UI

**Description**
I'm looking to hire a Thingsboard.io widget developer. I need a modified version of the Knob Control widget. With new graphics and modified behavior. 

If anyone is interested out there, please contact me.

**Comment by dhaval2017** [comment]
I can help 
contact me on dhaval@nistantritech.com

**Comment by tobbelin** [comment]
> **Component**
> 
> * UI
> 
> **Description**
> I'm looking to hire a Thingsboard.io widget developer. I need a modified version of the Knob Control widget. With new graphics and modified behavior.
> 
> If anyone is interested out there, please contact me.

Did you get any help with this? I have a similar problem needing a custom knob-control.

**Comment by thucar** [comment]
@tobbelin Yes, @dhaval2017 is working on a new widget for me. He is doing a great job at it too, I strongly suggest you get in contact with him

**Comment by Ultrazombie** [comment]
We inform you that your GitHub ticket has been inactive for some time, and we will close it. If you have any more concerns or encounter similar issues, please don't hesitate to contact us.
